react/__test__/Shelf.test.js

```diff
@@ -1,18 +1,56 @@
 /* eslint-env jest */
 import React from 'react'
-import { render } from 'test-utils'
+import { render } from '@vtex/test-tools/react'
 import ProductList from '../ProductList'
 
 describe('Shelf component', () => {
-  const renderComponent = () => {
+  const productMock = [
```

Comment:
Don't you think it's better if you extract this to other file?

Reply:
I didn't do this because its a small mock, but I will do it to keep the file smaller and easy to read